.vscode/launch.json

```diff
@@ -9,7 +9,18 @@
             "type": "python",
             "justMyCode": false,
             "request": "launch",
-            "name": "VehicleApp",
+            "name": "VehicleApp (native middleware)",
+            "program": "${workspaceFolder}/app/src/main.py",
+            "console": "integratedTerminal",
+            "env": {
+                "SDV_MIDDLEWARE_TYPE": "native"
```

Comment:
I've never been a fan of these launch configs which set the environment, it couples env vars between here and https://github.com/eclipse-velocitas/devenv-runtimes/pull/64/files#diff-44b589c17495145ec1695f5ff01fde5ea559684154dfb168437ee86e71da3099R36

We could switch this to use an envFile instead which is dynamically written by a preLaunchTask e.g. `velocitas exec runtime-local get-env`

Reply:
basically I agree that we have to many places where we define such variables, but we still need to give users the possibility to change it the way they need.

Reply:
As discussed we will keep this for time being. Will create a follow-up story for fixing this